__tests__/server/user-routes.spec.js

```diff
@@ -24,99 +25,53 @@ describe('Checks user answered routes', () => {
-  beforeAll(async () => {
-    const questionsRes = await request.get('/api/questions')
-    expect(questionsRes.status).toBe(200)
-    questions = questionsRes.body
-    questionIDs = questions.map(question => question.ID)
-    
-  })
-  // the following test might modify `answered` field of user test2
-  beforeEach(async () => {
-    clearFieldsInSingleRecord(userBaseName, user2Id, answeredFieldName)
-    await testSession.post('/api/auth/login').send({ Email: 'a2@b.com', Password: 'password'})
-    const res = await testSession.post('/api/user/answer').send({ questionId: questionIDs[7]})
-  })
-  it('should return updated user w/ new question in answered field', async () => {
-    res = await testSession.get('/api/user/answered')
-    expect(res.status).toBe(200)
-    expect(res.body.length).toBe(1)
-    console.log(res.body)
-    const answered = res.body//.Answered
-    expect(answered.includes(questions[7]))
-  }, 3000 ) // needs a little time for the change go through the DB
-  afterAll(async () => {
-    clearFieldsInSingleRecord(userBaseName, user2Id, answeredFieldName)
-  })
-})
-
-describe('Checks to see if user multiple answers update correctly', () => {
-  const answeredFieldName = 'Answered'
-  let questions
-  let questionIDs
-  let numOfQuestionsAdded
-  // get the question ids before the tests
-  beforeAll(async () => {
-    const questionsRes = await request.get('/api/questions')
-    expect(questionsRes.status).toBe(200)
-    questions = questionsRes.body
-    questionIDs = questions.map(question => question.ID)
-  })
-  // the following test might modify `answered` field of user test2
-  beforeEach(async () => {
-    clearFieldsInSingleRecord(userBaseName, user2Id, answeredFieldName)
-    await testSession.post('/api/auth/login').send({ Email: 'a2@b.com', Password: 'password'})
-
-    numOfQuestionsAdded = Math.min(4, questions.length)
+  it('should return updated ids of the answered questions when multiple questions are added sequentially of user test', async () => {
+    await testSession.post('/api/auth/login').send({ Email: 'a@b.com', Password: 'password'})
+    const numOfQuestionsAdded = Math.min(4, questions.length)
     for (let i = 0; i < numOfQuestionsAdded; ++i) {
       res = await testSession.post('/api/user/answer').send({ questionId: questionIDs[i]})
       expect(res.status).toBe(200)
     }
-  })
-  it('should return updated ids of the answered questions when multiple questions are added sequentially', async () => {
+
     res = await testSession.get('/api/user/answered')
     expect(res.status).toBe(200)
-    //expect(res.body.length).toBe(numOfQuestionsAdded)
-    expect(res.body.length).toBe(1)
+    expect(res.body.length).toBe(numOfQuestionsAdded)
     const answered = res.body
     answered.forEach((question, index) => {
       expect(question.ID == questions[index]) 
     }, 3000 )
   })
-})
+  it('should return updated user w/ new question in answered field', async () => {

```

Comment:
this test doesn't really do anything, so can you update it to actually call the answer route and answer a question for user test. then check that the size is 1. Also move it above the previous test since the previous one is testing answering multiple questions